src/WebSdk/Worker/Targets/Microsoft.NET.Sdk.Worker.targets

```diff
@@ -14,15 +14,24 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <EnableDefaultContentItems Condition=" '$(EnableDefaultContentItems)' == '' ">true</EnableDefaultContentItems>
     <RunWorkingDirectory Condition=" '$(RunWorkingDirectory)' == '' and '$(EnableDefaultRunWorkingDirectory)' != 'false' ">$(MSBuildProjectDirectory)</RunWorkingDirectory>
+    <MSBuildWorkerTargetsPath>$(MSBuildExtensionsPath)\Microsoft\VisualStudio\Managed.Worker\</MSBuildWorkerTargetsPath>
   </PropertyGroup>
 
-  <ItemGroup>
-    <ProjectCapability Include="DotNetCoreWorker" />
-    <ProjectCapability Include="SupportHierarchyContextSvc" />
-    <ProjectCapability Include="DynamicDependentFile" />
-    <ProjectCapability Include="DynamicFileNesting" />
-    <ProjectCapability Include="LocalUserSecrets" />
-    <ProjectCapability Include="WebNestingDefaults" />
-  </ItemGroup>
+  <Import Project="$(MSBuildWorkerTargetsPath)Microsoft.Worker.Designtime.targets" 
+         Condition="Exists('$(MSBuildWorkerTargetsPath)Microsoft.Worker.Designtime.targets')" />
+
+  <Choose>
+    <When Condition="!Exists('$(MSBuildWorkerTargetsPath)Microsoft.Worker.Designtime.targets')" >
```

Comment:
I don't think we should condition these on the targets file. Other IDE's could rely on these as well

Reply:
I was thinking this might give us a way to remove capabilities from VS if we need but your point is valid. I will remove the condition.